srcs/commands/strategy.c

```diff
@@ -17,10 +17,6 @@ void	run_strategy(t_cmds *cmds)
 {
 	if (ft_strcmp(cmds->current->type, "WORD") == 0)
 	{
-		if (cmds->current->next && cmds->current->strategy != S_RECEIVER)
```

Comment:
@carlosrocha-dev  the first part of command `export | grep SH` down here and this is wrong, just because it is different from `strategy != S_RECEIVER` in if...